### ISSUE TITLE
Octave installed via flatpak is not found

### ISSUE THREAD

**Issue by bagustris** [blink1073/oct2py#323]
The newest Octave usually is easier to install via [Flatpak](https://flathub.org/apps/org.octave.Octave), but using this way `oct2py ` cannot recognize the installed Octave. 

I also already my alias `octave` to point octave from flathub and it can called directly from terminal using `octave` command.

```python
$ ipython
Python 3.8.10 (default, Nov 22 2023, 10:22:35) 
Type 'copyright', 'credits' or 'license' for more information
IPython 8.10.0 -- An enhanced Interactive Python. Type '?' for help.

In [1]: import oct2py
octave not found, please see README

In [2]: 
```

Is there any step to link the octave installed from Flatpak to this oc2py? 
I am using Ubuntu 20.4 with the latest Octave version 9.2.0

**Comment by blink1073** [comment]
Hi @bagustris, have you tried setting `OCTAVE_EXECUTABLE`?

**Comment by bagustris** [comment]
There is no information how to set `OCTAVE_EXECUTABLE`. So, I assumed the following thing and it didn't work

```bash
(asd) bagus:~$ OCTAVE_EXECUTABLE='/var/lib/flatpak/exports/bin/org.octave.Octave' 
(asd) bagus:~$ python
Python 3.9.19 | packaged by conda-forge | (main, Mar 20 2024, 12:50:21) 
[GCC 12.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import oct2py
octave not found, please see README
>>>
```

**Comment by blink1073** [comment]
From the [README](https://github.com/blink1073/oct2py?tab=readme-ov-file#installation), "Alternatively, you can set an OCTAVE_EXECUTABLE or OCTAVE environment variable that points to octave executable itself.".  Is `/var/lib/flatpak/exports/bin/org.octave.Octave` an executable file?

**Comment by bagustris** [comment]
Yes, it is executable. I can run octave cli mode from that file.

**Comment by blink1073** [comment]
I'm not sure then, I'm not familiar with flatpack.
